transport/http/config.go

```diff
@@ -118,28 +119,61 @@ func (ts *transportSpec) buildInbound(ic *InboundConfig, t transport.Transport,
 
 // OutboundConfig configures an HTTP outbound.
 //
-// 	outbounds:
-// 	  keyvalueservice:
-// 	    http:
-// 	      url: "http://127.0.0.1:80/"
+//  outbounds:
+//    keyvalueservice:
+//      http:
+//        url: "http://127.0.0.1:80/"
 //
 // The HTTP outbound supports both, Unary and Oneway transport types. To use
 // it for only one of these, nest the section inside a "unary" or "onewy"
 // section.
 //
-// 	outbounds:
-// 	  keyvalueservice:
-// 	    unary:
-// 	      http:
-// 	        url: "http://127.0.0.1:80/"
+//  outbounds:
+//    keyvalueservice:
+//      unary:
+//        http:
+//          url: "http://127.0.0.1:80/"
+//
+// An HTTP outbound can also configure a peer list.
+// In this case, there can still be a "url" and it serves as a template for the
+// HTTP client, expressing whether to use "http:" or "https:" and what path to
+// use. The address gets replaced with peers from the peer list.
+//
+//  outbounds:
+//    keyvalueservice:
+//      unary:
+//        http:
+//          url: "https://address/rpc"
+//          round-robin:
+//            peers:
+//              - 127.0.0.1:8080
+//              - 127.0.0.1:8081
 type OutboundConfig struct {
+	config.PeerListConfig
+
 	// URL to which requests will be sent for this outbound. This field is
 	// required.
 	URL string `config:"url,interpolate"`
 }
 
 func (ts *transportSpec) buildOutbound(oc *OutboundConfig, t transport.Transport, k *config.Kit) (*Outbound, error) {
-	return t.(*Transport).NewSingleOutbound(oc.URL, ts.OutboundOptions...), nil
+	x := t.(*Transport)
```

Comment:
My reasoning is that "x" is sometimes pronounced "trans", as in TCP "xmit" and the yuletide holiday "Transmas".